lib/generators/connector/templates/home_controller.rb

```diff
@@ -7,46 +8,35 @@ def index
   def update
     organization = Maestrano::Connector::Rails::Organization.find_by_id(params[:id])
 
-    if organization && is_admin?(current_user, organization)
-      old_sync_state = organization.sync_enabled
-
-      organization.synchronized_entities.keys.each do |entity|
-        organization.synchronized_entities[entity] = !!params["#{entity}"]
-      end
-      organization.sync_enabled = organization.synchronized_entities.values.any?
-
-      unless organization.historical_data
-        historical_data = !!params['historical-data']
-        if historical_data
-          organization.date_filtering_limit = nil
-          organization.historical_data = true
-        else
-          organization.date_filtering_limit ||= Time.now
-        end
-      end
-      
-      organization.save
-
-      if !old_sync_state && organization.sync_enabled
-        Maestrano::Connector::Rails::SynchronizationJob.perform_later(organization, {})
-        flash[:info] = 'Congrats, you\'re all set up! Your data are now being synced'
-      end
+    return unless organization && is_admin?(current_user, organization)
```

Comment:
I'm being paranoid, but can double check that the spec are testing failing cases (no id, not admin) and are green (for all the methods of this controller)